src/octoprint/plugins/virtual_printer/virtual.py

```diff
@@ -246,6 +256,9 @@ def _reset(self):
             self._selectedSdFileSize = None
             self._selectedSdFilePos = None
 
+            # read eeprom from disk
+            self._virtual_eeprom.read_settings()
```

Comment:
This will run into an exception if virtual eeprom is disabled, in which case `self._virtual_eeprom` defaults to `None`. A simple `if self._virtual_eeprom` wrapper should suffice.